sos/report/plugins/containerd.py

```diff
@@ -19,6 +19,7 @@ class Containerd(Plugin, RedHatPlugin, UbuntuPlugin, CosPlugin):
     def setup(self):
         self.add_copy_spec([
             "/etc/containerd/",
+            "/etc/cni/net.d/",
```

Comment:
Just to be on the safe side - we don't have any data to obfuscate in the config files in that directory, right?

Reply:
only ips and network  parameters
```
 cat /etc/cni/net.d/10-kindnet.conflist

{
        "cniVersion": "0.3.1",
        "name": "kindnet",
        "plugins": [
        {
                "type": "ptp",
                "ipMasq": false,
                "ipam": {
                        "type": "host-local",
                        "dataDir": "/run/cni-ipam-state",
                        "routes": [
                               { "dst": "::/0" }
                        ],
                        "ranges": [
                                [ { "subnet": "fd00:10:244::/64" } ]
                        ]
                }
                ,
                "mtu": 1500
        },
        {
                "type": "portmap",
                "capabilities": {
                        "portMappings": true
                }
        }
        ]
}

```

the spec does not have credentials or something like this https://github.com/containernetworking/cni/blob/main/SPEC.md#example-configuration

Reply:
Perfect then, thank you!